ml-agents/mlagents/trainers/learn.py

```diff
@@ -82,6 +82,9 @@ def run_training(run_seed: int, options: RunOptions) -> None:
         )
         # Make run logs directory
         os.makedirs(run_logs_dir, exist_ok=True)
+        # Load any needed states
+        if checkpoint_settings.resume:
+            StatsReporter.load_state(os.path.join(run_logs_dir, "training_status.json"))
```

Comment:
Move `"training_status.json"` to a constant?

Reply:
I've moved it to a constant and the method to a helper method similar to the timing tree and configuration file writes.